packages/scheduler-prisma/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@steveojs/scheduler-prisma",
-  "version": "6.0.1",
+  "version": "6.0.2-beta",
```

Comment:
did you publish it to test locally?

we should move away from doing that and just using `yarn link`

Reply:
Noted

Reply:
```suggestion
  "version": "6.0.2",
```

@VLDCNDN this needs to drop the `-beta`